scripts/reorg.fish

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env fish
```

Comment:
We don't necessarily need to check this in at the end, though it may be useful as we iterate.